src/plugins/kibana/public/settings/sections/indices/partials/_edit_field_type.html

```diff
@@ -0,0 +1,13 @@
+<select ng-if="knownFieldTypes[field.name] !== 'geo_point'" name="field_type" ng-model="field.type">
```

Comment:
Yeah, or on the flip side I see I can convert bytes to geo_point for example.  How are the rules defined for which options are available?